spec/support/requests/api_helper.rb

```diff
@@ -0,0 +1,24 @@
+module APIHelper
+  def authenticated_api_get(path, token: nil)
+    get path, headers: api_headers(token:)
+  end
+
+  def authenticated_api_post(path, token: nil)
+    post path, headers: api_headers(token:)
+  end
+
+  def authenticated_api_put(path, token: nil)
+    put path, headers: api_headers(token:)
+  end
+
+  def api_headers(token: nil)
+    {
+      Authorization: "Bearer #{token || generate_api_token.token}"
+    }
+  end
```

Comment:
I was planning on adding the `rspec-default_http_header` gem for headers like npq/ecf1.
What do you think?

Reply:
maybe we hold off for now, seems to be cleanly done?

Reply:
Yeah I'm not sure its worth it for the use case we have at the moment. Happy to revisit if this gets out of hand though

Reply:
Sounds good, will do without gem 👍